src/code/components/clutch-view.js

```diff
@@ -40,48 +29,49 @@ const ClutchView = React.createClass({
     if (this.props.onClick) this.props.onClick(index, id, org);
   },
 
-  handlePageForward() {
-    this.setState({pagesBack: Math.max(this.state.pagesBack - 1, 0)});
+  handlePageBackward() {
+    this.setState({page: Math.max(this.state.page - 1, 1)});
   },
 
-  handlePageBackward() {
+  handlePageForward() {
     const numPages = this.props.orgs.length / 8;
-    this.setState({pagesBack: Math.min(this.state.pagesBack + 1, numPages - 1)});
+    this.setState({page: Math.min(this.state.page + 1, numPages)});
   },
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.orgs.length === 0) {
-      this.setState({pagesBack: 0});
+      this.setState({page: 0});
     }
     if (nextProps.orgs.length > this.props.orgs.length) {
       // Move to the end whenever a new clutch is bred
-      this.setState({pagesBack: 0});
+      const numPages = nextProps .orgs.length / 8;
+      this.setState({page: numPages});
     }
   },
 
   render() {
-    let displayStyle = {size: this.props.height*(5/8), top: -153, marginLeft: -12},
-        stableDrakeViews = this.props.orgs.map((org, index) => {
+    let displayStyle = {size: 136, top: -153, marginLeft: -12},
+        drakeViews = this.props.orgs.map((org, index) => {
```

Comment:
I fixed this now, by going back and editing earlier commits. When you look for the changes though, I suppose you'll just look at the final version? Normally I would make a "code review" commit, which is ugly, but does have the advantage of being easy for you to review.